tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -102,6 +102,11 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
                         string s => new HashSet<string>(s.Split([','], StringSplitOptions.RemoveEmptyEntries)),
                     };
 
+            PropagateProcessTags = config
+                                       .WithKeys(ConfigurationKeys.PropagateProcessTags)
+                                       .AsBool(false)
+                                || ExperimentalFeaturesEnabled.Contains(ConfigurationKeys.PropagateProcessTags);
```

Comment:
Is this really correct? 🤔 If the customer explicitly sets `DD_EXPERIMENTAL_PROPAGATE_PROCESS_TAGS_ENABLED=false`, but they happen to have "all" for experimental features, then they have no way to disable the feature? Should it actually be that the _default_ value is controlled by that? 

```suggestion
                                       .AsBool(
                                       ExperimentalFeaturesEnabled.Contains(ConfigurationKeys.PropagateProcessTags);
```